webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -66,14 +66,31 @@ test.describe('Experiment List', () => {
         await projectDetailsPage.f_experimentList.tableActionBar.columnPickerMenu.open();
       await columnPicker.columnPickerTab.reset.pwLocator.click();
       await columnPicker.close();
+      await waitTableStable();
     });
     await test.step('Reset Filters', async () => {
       const tableFilter =
         await projectDetailsPage.f_experimentList.tableActionBar.tableFilter.open();
       await tableFilter.filterForm.clearFilters.pwLocator.click();
       await tableFilter.close();
+      await waitTableStable();
+    });
+    await test.step('Sort Oldest → Newest', async () => {
+      // reset
+      const content = await projectDetailsPage.f_experimentList.tableActionBar.multiSortMenu.open();
+      await content.multiSort.reset.pwLocator.click();
+      // the menu doesn't close in automation, but it works with mouse events manually
+      // await projectDetailsPage.f_experimentList.tableActionBar.multiSortMenu.open();
+
+      // set sort
+      const firstRow = content.multiSort.rows.nth(0);
+      await firstRow.column.selectMenuOption('Start time');
+      await firstRow.order.selectMenuOption('Oldest → Newest');
+      // [ET-284] wait before popover close or else it'll flake
+      await authedPage.waitForTimeout(1_000);
```

Comment:
i'm wondering if there's a better way to wait for the page to settle other than adding a timeout. Menu state is debounced to keep the UI responsive for users -- while we're planning a fix to ensure the menu state is committed on close, [waiting for a matching request](https://playwright.dev/docs/api/class-page#page-wait-for-request) should decrease the amount of time that the test is waiting.

Reply:
since the fix is planned, i think i'd rather just wait for the debounce to end for now. it'll be easy to delete the line later once the work gets done. i've got other stuff i need to focus on for now. thanks for approving

Reply:
i mean, it would also work for the current `waitTableStable` stuff as well.

Reply:
i just added it to my backlog. thanks for the suggestion